### ISSUE TITLE
Form rendering issues with theme config being stored in project config

### ISSUE THREAD

**Issue by chasegiunta** [verbb/formie#1110]
### Describe the bug

It seems like there's some issues with theme config changes being propagated correctly to project config. I was originally just trying to add a css class to the field's wrapper using Formie's field settings, but I wasn't getting the css class applied, even after clearing all caches. Curiously, I commented out my entire `formie.php` config file, cleared caches, and still was getting the same exact output. I removed the entire file. Still, same output.

After searching my project, I realized that Formie was storing the theme config in the project config, and this wasn't being correctly updated. From the outside looking in, it _does_ seem a bit unintuitive to store this in project config.

It was only after _redeclaring_ the `themeConfig` option in my formie config that the forced formie to reset it's render output correctly:

```
<?php

return [
  '*' => [
    'themeConfig' => [],
  ],
];
```

### Steps to reproduce

1. Use `formie.php` to create a theme config.
2. Create a form.
3. Create a field or two.
4. Go ahead and render that form in a template via barebones `craft.formie.renderForm('testForm)` and see it working.
5. Back in the form settings, add an attribute or custom class to the field wrapper element.
6. Refresh your form, observe no changes.
7. Comment out your entire formie.php file, leaving the file itself.
8. Refresh form template, see the same changes.
9. Delete formie config file
10. Refresh form template, see the same changes.
11. Add back in formie config file, with empty themeconfig value:
```
  '*' => [
    'themeConfig' => [],
  ]
```
12. Refresh form template, see all is rendering correctly.

### Form settings

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: No
- Custom Form Templates: No

### Craft CMS version

4.2.5.1

### Plugin version

2.0.12

### Multi-site?

No

**Comment by engram-design** [comment]
Yeah, the theme config shouldn't be in project config entirely! Fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.12",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.13](https://github.com/verbb/formie/releases/tag/2.0.13)
